### PR TITLE
Create function to calculate the integral using  simpson.

### PR DESCRIPTION
**Pull request by Priyanka-43**
For this assignment all pull requests going  forward should contain the following (please delete the placeholder text and add your own description)

# Title
  - A ledgible title that briefly describes the contents of the pull request.

## Linting
  - Does your code lint? If not are the errors within acceptable errors?

## Contents
  - A legible description that indepthly describes the contents of the pull request.
  - A list of files that are new / changed
  - The contents that you have been assigned to submit labeled with titles that acuratly describe the assignment they are for

## Labels and Milestones
  - ALL pull requests must have an attached label and Milestone before a reviewer can be assigned.

## Reviewer
  - add a sugggested reviewer here (perhaps your team mate)

Any further questions on formatting pull requests can be asked in the issues tab or on discord. 
This document can be changed at any time.

### CODE REVIEW

**Comment by zbpetersbuf** [comment]
@Priyanka-43 this pull request still needs a reviewer, also you will need to update your branch before I can merge it

File: calculus.py [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+    a (float): The start of the interval.
+    b (float): The end of the interval.
+    n (int): The number of intervals (must be even). Default is 100.
+
+    Returns:
+    float: The approximate integral of f over [a, b].
+    """
+    h = (b - a) / n
+    i = np.arange(0, n)
+    # Ensure n is even
```
[the remark this answers is not in the document]
Comment (Priyanka-43) [reply calculus.py:28]: I have made changes. But i am unable to see my pylint and pytest.

[the remark this answers is not in the document]
Comment (zbpetersbuf) [reply calculus.py:32]: you should be able to run pylint and pytest in your module through your terminal no need to push each time you change something

[the remark this answers is not in the document]
Comment (zbpetersbuf) [reply calculus.py:32]: what you do is in terminal type cd (module name) so in this case
cd calculus.py
the press return then type
pylint calculus.py

[the remark this answers is not in the document]
Comment (zbpetersbuf) [reply calculus.py:32]: for pytest,
you will before you cd calculus.py,

you type cd test_calculus.py
then type pytest then your function name so in terminal type

cd cd test_calculus.py
then press return

pytest test_calculus.py
then press return (im pretty sure, its either the file or the test function name but I think its the file name)

**Comment by Priyanka-43** [comment]
Hi, I have made the required changes. If you can merge this pull request.

**Comment by zbpetersbuf** [comment]
looks like you resolved @laserlab issue

**Review by zbpetersbuf — APPROVED** [review]

**Pull request merged**